tests/acceptance/features/webUIPrivateLinks/accessingPrivateLinks.feature

```diff
@@ -18,4 +18,22 @@ Feature: Access private link
     Given the user navigates to the private link created by user "user1" for file "lorem.txt"
     When user "user1" has logged in using the webUI
     Then the app-sidebar for file "lorem.txt" should be visible on the webUI
-    And the "versions" details panel should be visible
\ No newline at end of file
+    And the "versions" details panel should be visible
+
+  Scenario: Access private link as a collaborator
+    Given user "user2" has been created with default attributes
+    And user "user1" has shared file "lorem.txt" with user "user2" with "read" permissions
+    And user "user2" has logged in using the webUI
+    When the user navigates to the private link created by user "user1" for file "lorem.txt"
+    Then the app-sidebar for file "lorem (2).txt" should be visible on the webUI
+    And the "versions" details panel should be visible
+
+  Scenario: Access the private link as an user that does not have permissions to see the file
+    Given user "user2" has been created with default attributes
+    And user "user2" has logged in using the webUI
+    When the user navigates to the private link created by user "user1" for file "lorem.txt"
+    Then the user should see the following error message on the login card dialog
```

Comment:
There should be a better error please raise issue for this and tag it or check if there is already an issue.